tests/contracts/contracts/migration/StakingEscrowOld.sol

```diff
@@ -1630,63 +1630,7 @@ contract StakingEscrowOld is IssuerOld, IERC900History {
     /// @dev the `onlyWhileUpgrading` modifier works through a call to the parent `verifyState`
     function verifyState(address _testTarget) public override virtual {
```

Comment:
Same question here — if it just calls the superclass, does it need to be overridden at all?

Reply:
✔️